networking/src/p2p/peer.rs

```diff
@@ -298,7 +460,9 @@ impl Actor for Peer {
             }, None);
 
             // begin to process incoming messages in a loop
-            begin_process_incoming(net, myself.clone(), network_channel, log).await;
+            begin_process_incoming(net, myself.clone(), network_channel, throttle_quota, log.clone()).await;
+
+            notify.notify_one();
```

Comment:
@akoptelov 
I am not sure if this `notify.notify_one();` is trigger all the time when peer craches, thread fails...
maybe, I would add also another `notify.notify_one();`  to the `post_stop` just to be sure

Reply:
I've just moved it there.